lib/ansible/modules/monitoring/zabbix/zabbix_proxy.py

```diff
@@ -36,12 +36,18 @@
     - "Alen Komic (@akomic)"
 requirements:
     - "python >= 2.6"
-    - "zabbix-api >= 0.5.4"
+    - "zabbix-api >= 0.5.3"
 options:
     proxy_name:
         description:
             - Name of the proxy in Zabbix.
         required: true
+    proxy_address:
+        description:
+            - Address of the active proxy in Zabbix to limit access ( Zabbix Version >= 4.0.0 )
+            - Remove option for Zabbix Version < 4.0.0
```

Comment:
This should be documented more clearly , I would propose something like this to align with other zabbix modules:

```yml
        description:
            - Comma-delimited list of IP/CIDR addresses or DNS names to accept active proxy requests from.
            - Requires I(status=active).
            - Works only with >= Zabbix 4.0.
```

Reply:
Looks good - will change. I will just additionally mention the option remove:

`- Works only with >= Zabbix 4.0. ( remove option for <= 4.0 )`